Jenkinsfile

```diff
@@ -61,8 +63,16 @@ pipeline {
                     git lfs pull
                     git lfs checkout
                     cd $curdir
+                    export api_branch=$(git config -f .gitmodules submodule.pmm-api-tests.branch)
```

Comment:
It would be much better if we use submodule commit, not just branch. Is it possible?

Reply:
I don't think it would be helpful, because our API tests job uses branch parameter and then checks out the branch we provide to run tests